scripts/renovate/compareImagesAndCharts.ts

```diff
@@ -231,10 +231,10 @@ function compareImages(oldImages: any, newImages: any, result: ComparisonResult)
               result.labels.push('waiting on ironbank');
               result.changes.push(`Waiting on Ironbank image update: ${missingImg}`);
             }
-          } else if (missingImg.includes('cgr.dev')) {
-            if (!result.labels.includes('waiting on cgr')) {
-              result.labels.push('waiting on cgr');
-              result.changes.push(`Waiting on Chainguard image update: ${missingImg}`);
+          } else if (missingImg.includes('quay.io')) {
```

Comment:
My poor code here but we may want to change this to `startsWith` and make sure that this only catches RF images from quay.
```suggestion
          } else if (missingImg.startsWith('quay.io/rfcurated')) {
```

It might not be required but makes sure this logic wouldn't catch stuff like keycloak/prometheus which host their upstreams on quay as well.